app/models/invoice.rb

```diff
@@ -25,6 +25,7 @@ class Invoice < ApplicationRecord
 
   belongs_to :organization, inverse_of: :invoices
   belongs_to :customer, inverse_of: :invoices
+  belongs_to :bank_account, optional: true
```

Comment:
@yunixon add inverse transaction with `dependent: :destroy` and add `inverse_of`